org.ghidra_sre.Ghidra.json

```diff
@@ -34,56 +34,73 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://savannah.nongnu.org/download/icoutils/icoutils-0.31.1.tar.bz2",
-                    "sha256": "26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+                    "sha256": "17abe02d043a253b68b47e3af69c9fc755b895db68fdc8811786125df564c6e0",
+                    "url": "https://download.savannah.gnu.org/releases/icoutils/icoutils-0.32.3.tar.bz2",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 1360,
+                        "url-template": "https://download.savannah.gnu.org/releases/icoutils/icoutils-$version.tar.bz2"
+                    }
                 }
             ]
         },
         {
             "name": "ghidra",
             "buildsystem": "simple",
             "build-commands": [
-                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
-                "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
-                "icotool -x support/ghidra.ico --index=8",
-                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
-                "install -Dm644 org.ghidra_sre.Ghidra.desktop /app/share/applications/org.ghidra_sre.Ghidra.desktop",
-                "install -Dm644 org.ghidra_sre.Ghidra.metainfo.xml /app/share/metainfo/org.ghidra_sre.Ghidra.metainfo.xml",
-                "install -Dm755 ghidra.sh /app/bin/ghidra",
-                "mv /app/lib/ghidra/support/launch.properties{,.orig}",
-                "ln -s /var/config/ghidra.properties /app/lib/ghidra/support/launch.properties",
-                "sed -i 's,bg,fg,' /app/lib/ghidra/ghidraRun"
+                "sh -e build.sh",
+                "unzip build/dist/ghidra_*_FLATPAK_*_linux_*.zip",
+                "icotool -x Ghidra/RuntimeScripts/Windows/support/ghidra.ico -i 8",
+                "cp -r ghidra_*_FLATPAK /app/ghidra",
+                "mkdir -p /app/share/applications",
+                "cp org.ghidra_sre.Ghidra.desktop /app/share/applications",
+                "mkdir -p /app/share/icons",
+                "cp ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
+                "mkdir -p /app/share/appdata",
+                "cp org.ghidra_sre.Ghidra.appdata.xml /app/share/appdata",
+                "ln -s /app/ghidra/ghidraRun /app/bin/ghidra"
             ],
+            "build-options": {
+                "build-args": [
+                    "--share=network"
```

Comment:
@hadess This is solving the network issue

Reply:
Wait... Ok I get it. We can't change the build options: `Can't specify build-args in sandboxed build`

Reply:
This isn't something I can merge...

Reply:
I'll try to use another method.